### PR TITLE
 Let the "owner" be translated 

### PR DESCRIPTION
**Pull request by kwl01skz**
The owner of  item and vehicle  is not displayed.
Just need to put them into gettext()

 ```SUMMARY: I18N "Let the owner be translated "```

#### Describe alternatives you've considered
I am not sure, maybe we can translate them at an earlier time?

### CODE REVIEW

File: lang/extract_json_strings.py [hunk 1]
```diff
@@ -205,6 +205,7 @@ def extract_harvest(item):
 
 def extract_bodypart(item):
     outfile = get_outfile("bodypart")
+    writestr(outfile, item["name"])
```
Comment (kevingranade) [inline lang/extract_json_strings.py]: Unless I'm missing something this is unrelated to the rest of the PR.
it's also redundant with the next line.

Comment (kevingranade) [inline lang/extract_json_strings.py]: Unless I'm missing something this is unrelated to the rest of the PR.
it's also redundant with the next line.

Comment (kwl01skz) [reply lang/extract_json_strings.py]: I thought so too, so I deleted this line in #32655  , but not long after I found that the translation of the "body part" in the "effects" disappeared, and the translation of the "body part" in the "clothes" was still there.
After querying I found this code.

```
std::string body_part_name( body_part bp, int number )
{
    const auto &bdy = get_bp( bp );
    return ngettext( bdy.name.c_str(),
                     bdy.name_multiple.c_str(), number );
}

std::string body_part_name_accusative( body_part bp, int number )
{
    const auto &bdy = get_bp( bp );
    return npgettext( "bodypart_accusative",
                      bdy.name.c_str(),
                      bdy.name_multiple.c_str(), number );
}
```

The author creates two similar functions, both of which are used for the return bodypart, providing the same msgid, but providing context for one of them, context = "bodypart_accusative".


I guess the "body part" of some languages ​​is not the same as using accusative and nominative.

If we need to simplify, deleting the second sentence should not cause problems, at least in Chinese and English.

```
    writestr(outfile, item["name"])
    writestr(outfile, item["name"], context="bodypart_accusative")
```

Comment (kevingranade) [reply lang/extract_json_strings.py]: I see, they're the same in English but might not be in other languages, thanks for the correction.
